internal/configconverter/config_server.go

```diff
@@ -156,26 +159,21 @@ func (cs *ConfigServer) start() {
 
 			go func() {
 				httpErr := cs.server.Serve(listener)
+				defer cs.serverShutdown.Done()
 				if httpErr != http.ErrServerClosed {
```

Comment:
can httpErr be nil?

Reply:
From the docs: `Serve` always returns a non-nil error and closes l. After Shutdown or Close, the returned error is `ErrServerClosed`.